hw2/02-cssgrid.md

```diff
@@ -0,0 +1,32 @@
+## Exercise 02 - CSS Grid
+
+1. grid-column-start: 3;
+2. grid-column-start: 5;
+3. grid-column-end: 4;
+4. grid-column-end: 2;
+5. grid-column-end: -2;
+6. grid-column-start: -3;
+7. grid-column-end: span 2;
+8. grid-column-end: span 5;
+9. grid-column-start: span 3;;
+10. grid-column: 4 / 6;
+11. grid-column: 2/ span 3;
+12. grid-row-start: 3;
+13. grid-row: 3/ span 3;
+14. grid-column: 2;
+    grid-row: 5;
+15. grid-column: 2/ span 4;
+    grid-row: span 5;
+16. grid-area: 1/ 2 / 4 / 6;
+17. grid-area: 2/ 3/ 5/ 6;
+18. order: 4;
+19. order: -1;
+20. grid-template-columns: 50% 25% 25%;
+21. grid-template-columns: repeat(8, 12.5%);
+22. grid-template-columns: 100px 3em 40%;
+23. grid-template-columns: 1fr 5fr;
+24. grid-template-columns: 50px 1fr 1fr 1fr 50px;
+25. grid-template-columns: 75px 3fr 2fr;
+26. grid-template-rows: repeat(4, 12.5px);
```

Comment:
I did it with grid-template-rows: 50px 0fr 0fr 0fr;

Reply:
Yeah I struggled a lot with this one